src/stories/Autocomplete.stories.tsx

```diff
@@ -740,6 +752,54 @@ export const InOverlayWithCustomScrollContainerRef = (args: FormControlArgs<Auto
   )
 }
 
+export const NestedInsideOverlay = (args: FormControlArgs<AutocompleteArgs>) => {
```

Comment:
I'm not able to move focus between the inputs using the `Tab` key.

Reply:
Okay, good to know.